gradle/multiNodeTesting.gradle

```diff
@@ -8,6 +8,11 @@ task testMultiNode(type: Exec) {
     environment "BUILD_DIR", project.buildDir
 
     def args = ['bash', './testMultiNode.sh']
+
+    if(project.hasProperty('test_ssl')) {
```

Comment:
It is better to pass it via environment variable. The same way as `BUILD_DIR` above. Just to have consistent code.